src/main/java/mate/academy/model/Movie.java

```diff
@@ -9,6 +9,7 @@
 @Entity
 @Table(name = "movies")
 public class Movie {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
```

Comment:
For each entity, you use the same identifier type.
How about extracting it to `AbstractEntity` class?